src/dan2.js

```diff
@@ -122,6 +120,8 @@ const register = function(url, params, callback) {
             _i_chans.add('ctrl', metadata['ctrl_chans'][0]);
             _o_chans.add('ctrl', metadata['ctrl_chans'][1]);
             _rev = metadata['rev'];
+            _mqtt_host = metadata.url['host'];
+            _mqtt_port = metadata.url['ws_port'];
```

Comment:
Is the scope of `_mqtt_host` valid for following usage?
I'm not quite sure how JS's scope works.

Reply:
Yes, it's valid